src/reducers/index.js

```diff
@@ -74,7 +70,7 @@ const rootReducer = (state = initialState, action) => {
     case 'reload_data':
```

Comment:
*comment partially related to the line number*

I suggest using immerjs (https://immerjs.github.io/immer/) for state mutations

Maybe not on this PR, but I like to use immer to make sure that our state only mutates on reducers (that use immer)

Reply:
Humm I like this package and I've never seen it. Will create an issue to add this to the project in future improvements.